### PR TITLE
[Fix] Calculate taxes when shipping rule not present at item removed.

### PR DESCRIPTION
**Pull request by shreyashah115**
#12583 
Issue:
items_remove_error
Fix:
items_remove_fix

### CODE REVIEW

File: erpnext/public/js/controllers/transaction.js [hunk 1]
```diff
@@ -648,6 +648,9 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 				}
 			}).fail(() => this.frm.set_value('shipping_rule', ''));
 		}
+		else {
```
Comment (vishdha) [inline erpnext/public/js/controllers/transaction.js]: No need to call calculate_taxes_and_totals here.

**Comment by vishdha** [comment]
In your first image.  _Test_FG_Item may not contain weight details. Add weight_per_unit at item master.

**Review by vishdha — APPROVED** [review]

**Comment by vishdha** [comment]
change base from develop to hotfix.

**Comment by shreyashah115** [comment]
Will create a new PR.

**Pull request closed without merging**